core/src/test/java/io/questdb/griffin/engine/functions/geohash/GeoHashNativeTest.java

```diff
@@ -95,4 +98,61 @@ public void testSlideFoundBlocks() {
         }
 
     }
+
+    @Test
+    public void testLatestByAndFilterPrefixShouldNotAccessUnmappedMemory() {
+        Path path = new Path().of(configuration.getRoot());
+
+        // allocate and map 1-page index
+        long pageSize = Files.PAGE_SIZE;
+        int N = (int) ((pageSize - 64) / 32);
+        int keyCount = N + 1; // +1 is important here
+
+        final DirectLongList rows = new DirectLongList(keyCount, MemoryTag.NATIVE_LONG_LIST);
+        long argsAddress = LatestByArguments.allocateMemoryArray(1);
+
+        try {
+            BitmapIndexTest.create(configuration, path, "x", 64);
+
+            rows.setCapacity(keyCount);
+            GeoHashNative.iota(rows.getAddress(), rows.getCapacity(), 0);
+
+            final long klo = 0;
+            final long khi = keyCount;
```

Comment:
nit: there is IDE warning regarding redundant variable here

Reply:
Fixed in: https://github.com/questdb/questdb/pull/3071/commits/0abd432ad52d97f59a592ccce9905e9285bf4199

Reply:
thanks 👍